canonical/juju-dashboard-charm
# WD-3493 - Add Juju 3.2 support

## Changes
- Update Juju Dashboard to 0.11.1.


## QA
- Set up a Juju 3.2 controller.
- Follow the instructions for building the machine charm:
https://github.com/canonical/juju-dashboard-charm#building-and-testing-the-machine-charm
- Go to the controllers page and check that the controller(s) appear in the list.
- Go to the model list and click on a model, the details page should load.
- Click on the access button and check that the panel opens.
- Open your dev tools and check that there aren't any errors.


https://warthogs.atlassian.net/browse/WD-3493
